bundles/myelin-input-events/input-events/memorial.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+/**
+ * Account character selection event
+ */
+module.exports = (srcPath) => {
+  const EventUtil = require(srcPath + 'EventUtil');
+  const Data = require(srcPath + 'Data');
+  const humanizeDuration = require('humanize-duration');
+
+  return {
+    event: state => (socket, args) => {
+      let { name } = args;
+      const say = EventUtil.genSay(socket);
+      const write = EventUtil.genWrite(socket);
+
+      // Get pfile of deceased to show stats.
+      const deceased = Data.load('player', name);
+      const attrs = deceased.attributes;
+      const { killedOn, killedBy } = deceased.metadata;
+      const timeSinceDeath = killedOn ?
+        humanizeDuration(Date.now() - killedOn, { largest: 2, round: true }) :
+        'a long time';
+      const effects = deceased.effects;
+
+      const killerName = killedBy === name ? 'a lethal condition' : killedBy;
```

Comment:
If they die from an effect or an environmental damage, the deceased are the "killer". Not sure if there's a good way to address this in the rest of the code but for now I'm leaving this and adding a list of effects so players can suss out how they died.